orc8r/lib/go/initflag/initflag.go

```diff
@@ -10,58 +10,47 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-// package initflag initializes (parses) Go flag if needed, it allows the noise free use of golog & other packages
-// relying on flags being parsed
+
+// Package initflag parses a subset of flags, optionally redirecting output
+// to syslog.
 package initflag
 
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 )
 
-func init() {
-	flag.Var(
-		&syslogDest,
-		syslogFlag,
-		"Redirect stderr to syslog, optional syslog destination in network::address format (system default otherwise)")
+var (
+	flags          = flag.NewFlagSet("magma_tmp_flags", flag.ContinueOnError)
+	stdoutToStderr = flags.Bool("stdout2stderr", false, "Redirect stdout to stderr")
 
-	// only if not already parsed
-	if !flag.Parsed() {
```

Comment:
Try to replace this line with:

	// only if not already parsed and not a test build
	if !(flag.Parsed() ||
		strings.HasSuffix(os.Args[0], ".test") ||
		strings.HasSuffix(os.Args[0], "_test.go") ||
		strings.HasSuffix(os.Args[0], "_test_go")) {

it should solve the issue.